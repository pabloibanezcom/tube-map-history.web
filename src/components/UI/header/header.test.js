import React from 'react';
import Header from './header';

describe('header', () => {
  it('should render header', () => {
    const component = <Header />;

    expect(component).toMatchSnapshot();
  });
});