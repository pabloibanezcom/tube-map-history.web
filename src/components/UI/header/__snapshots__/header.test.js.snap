// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header should render header 1`] = `<header />`;
